LDM-592.tex

```diff
@@ -967,6 +967,18 @@ \section{Data Release Processing}
 
 }
 
+\usecase{DRP32}{Use of rerun collections}{%
+
+\item
+A Developer runs some processing starting from outputs produced by another Developer, where those outputs are specified as inputs using a label given as a single string (possibly with path-like namespace components).
+They specify a new label to use for the output data location.
+When the processing completes, all of the contents of the inputs (or at least all of the items actually used as inputs) will appear to be present in the output collection.
+
+\item
+The developer then wants to run additional processing with both inputs and outputs coming from what was previously just the output collection.
```

Comment:
I'm not sure that this is a different case.  If I decide that I want to do more processing I use `--rerun rhl/tmp` to write more outputs, and possibly classes of outputs, to rerun `rhl/tmp`.  Of course, `--rerun rhl/tmp:rhl/foo` (with a different configuration of options/versions) will work too if I want to compare the outputs in `rhl/tmp` to those in `rhl/foo`.

Reply:
Right; I think as it's currently formatted it's not actually a different use case.  I just wanted to make sure the use case captured the idea that the rerun implicitly contains the inputs (recursively), not just the outputs.

Note, however, that passing `--rerun rhl/tmp:rhl/foo` to a pipe_base argument parser does not compare outputs; it creates a new rerun `rhl/foo` for output that contains (via the parent mechanism, for now) everything in `rhl/tmp`.  If you want to do a comparison that involves looking at the same data ID in different repos you need two `Butlers` (and this is not planned to change with Gen3).